content/news/2025_02_11_mac_app_notarized.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "GRASS GIS App for Mac now notarized"
+date: 2025-02-11T15:00:00+00:00
+layout: "news"
+author: Nicklas Larsson
+---
+
+The GRASS Development Team is pleased to announce that the prebuilt
+applications for macOS are now signed and notarized. This enhancement
+represents a significant improvement in both security and user convenience.
+
+<img src="/images/news/2025_02_11_mac_notarized.png"
+     alt="GRASS GIS App for Mac now notarized"
+     title="GRASS GIS App for Mac now notarized"
+     width="45%" style="float:right">
+
+The currently available notarized binaries include the preview version
```

Comment:
Should there somewhere be a link to get the binaries?

Reply:
Don’t know if necessary, there is “Download” menu item above.